modules/json/src/parsers/convert-json-expression.js

```diff
@@ -0,0 +1,74 @@
+import {get} from '../utils/get';
+
+// expression-eval: Small jsep based expression parser that supports array and object indexing
+import expressionEval from 'expression-eval';
+
+class EvaluationError extends Error {
```

Comment:
In our code base we do not inherit from `Error` in any other place. 

Reasons are that `Error` is a built-in class, and I have run into "gotchas" in the past when inheriting from it (not sure if this is still an issue in the latest browsers/babel), but more importantly, as appealing as Error subclasses are to framework writers, I never see the need in applications to distinguish errors by type.

Suggest dropping this as the test cases can be handled without it...

Reply:
+1 certain browsers also forbid extending built-in classes.

Reply:
works for me, will remove